electron-app/src/store/balances/getters.ts

```diff
@@ -112,6 +114,76 @@ export const getters: GetterTree<BalanceState, RotkehlchenState> = {
     }, Zero);
   },
 
+  // simplify the manual balances object so that we can easily reduce it
+  manualBalanceByLocation: (
+    state: BalanceState,
+    balanceGetters,
+    rootGetters
+  ) => {
+    const fiatTotal = balanceGetters.fiatTotal;
+    const mainCurrency =
+      rootGetters.session?.settings.selectedCurrency.ticker_symbol;
+
+    const manualBalances = state.manualBalances;
+    const exchangeRate = balanceGetters.exchangeRate(mainCurrency);
```

Comment:
if you only use a couple of properties you should be able to destructure in the function call.